src/Tizen.NUI/src/public/Text/Spans/StrikethroughSpan.cs

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+
+namespace Tizen.NUI.Text.Spans
+{
+    /// <summary>
+    /// </summary>
+    // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class StrikethroughSpan : BaseSpan
+    {
+
+        /// <summary>
+        /// Create strikethrough span.<br />
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static StrikethroughSpan CreateStrikethrough()
+        {
+            return StrikethroughSpanBuilder.Initialize().Build();
+        }
+
+        internal StrikethroughSpan(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// <summary>
+        /// The strikethrough color in strikethrough-span.
+        /// </summary>
+        /// This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Color StrikethroughColor
```

Comment:
How about `Color` instead `StrikethroughColor` because it is StrikethroughSpan class. I think it duplicated of `Strikethrough`
```suggestion
        public Color Color
```
``` c#
 StrikethroughSpan span = StrikethroughSpan.Create();
 span.Color = Color.Red;
 // vs
 span.StrikethroughColor = Color.Red
```